dialogue-clients/src/test/java/com/palantir/dialogue/clients/ReloadingClientFactoryTest.java

```diff
@@ -91,4 +99,42 @@ void live_reloading_wrapper_still_uses_the_EndpointChannelFactory_factory() {
         // ensure we use the bind method
         verify(channel, atLeastOnce()).endpoint(any());
     }
+
+    @Test
+    void withConjureErrorParameterSerializationFormat_wraps_runtime_with_specified_format() {
+        ChannelCache mockCache = mock(ChannelCache.class);
+        ImmutableReloadingParams params = ImmutableReloadingParams.builder()
+                .scb(Refreshable.only(ServicesConfigBlock.builder().build()))
+                .build();
+        ReloadingClientFactory factory = new ReloadingClientFactory(params, mockCache);
+
+        // Get the original runtime
+        ConjureRuntime originalRuntime = params.runtime();
+        assertThat(originalRuntime.bodySerDe().errorParameterFormat()).isEqualTo(Optional.empty());
+
+        // Set the error parameter format to JSON
+        ReloadingFactory modifiedFactory =
+                factory.withConjureErrorParameterSerializationFormat(ConjureErrorParameterFormat.JSON_FORMAT);
+
+        // Call get
+        RuntimeCapturingService service = modifiedFactory.get(RuntimeCapturingService.class, "foo");
+
+        assertThat(service.runtime().bodySerDe().errorParameterFormat())
+                .contains(ConjureErrorParameterFormat.JSON_FORMAT);
```

Comment:
I would also verify that we didn't change anything on the original factory and runtime, provided that is our intent
```suggestion
        // Call get
        RuntimeCapturingService service = modifiedFactory.get(RuntimeCapturingService.class, "foo");

        assertThat(service.runtime().bodySerDe().errorParameterFormat())
                .contains(ConjureErrorParameterFormat.JSON_FORMAT);
        
        // Verify we didn't impact the original factory
        RuntimeCapturingService originalFactoryService = factory.get(RuntimeCapturingService.class, "foo");
        
        assertThat(originalFactoryService.runtime().bodySerDe().errorParameterFormat()).isEmpty();
```